modules/distributor/distributor.go

```diff
@@ -129,6 +129,7 @@ type Distributor struct {
 	generatorClientCfg      generator_client.Config
 	generatorsRing          ring.ReadRing
 	generatorsPool          *ring_client.Pool
+	generatorQueue          *forwarder
```

Comment:
nit: call this `generatorForwarder` since it's a forwader?

Reply:
Changed in 4b5aeb3ef729ddc22dfba07139911ff75cbc7b46